test/content-parser.test.js

```diff
@@ -10,43 +9,45 @@ const first = function (req, payload, done) {}
 const second = function (req, payload, done) {}
 const third = function (req, payload, done) {}
 
-test('hasContentTypeParser', t => {
-  test('should know about internal parsers', t => {
+test('hasContentTypeParser', async t => {
+  await t.test('should know about internal parsers', (t, done) => {
     t.plan(5)
 
     const fastify = Fastify()
     fastify.ready(err => {
-      t.error(err)
-      t.ok(fastify.hasContentTypeParser('application/json'))
-      t.ok(fastify.hasContentTypeParser('text/plain'))
-      t.ok(fastify.hasContentTypeParser('  text/plain  '))
-      t.notOk(fastify.hasContentTypeParser('application/jsoff'))
+      t.assert.ifError(err)
+      t.assert.ok(fastify.hasContentTypeParser('application/json'))
+      t.assert.ok(fastify.hasContentTypeParser('text/plain'))
+      t.assert.ok(fastify.hasContentTypeParser('  text/plain  '))
+      t.assert.ok(!fastify.hasContentTypeParser('application/jsoff'))
```

Comment:
nit: feel like this is easy to read in test file vs looking for `!` [ref](https://nodejs.org/api/assert.html#assertiferrorvalue)
```suggestion
      t.assert. ifError(fastify.hasContentTypeParser('application/jsoff'))
```

Reply:
I think that `hasContentTypeParser` returns `boolean` value and in that case `ifError` is not good for us.

Reply:
this should work if `boolean` is returned
```js
t.assert.strictEqaul(fastify.hasContentTypeParser('application/jsoff'), false)
```

Reply:
This could work... but I honestly dont think its better. if you insist ill change that...

Reply:
@dancastillo please let me know what you think